### PR TITLE
Adding ExampleObjects to the Parameters

### PR DESCRIPTION
**Pull request by xenkidu-va**
### [API-8480](https://vajira.max.gov/browse/API-8480)
We are adding ExampleObjects to the Parameters.

### CODE REVIEW

File: community-care-eligibility-api/src/main/java/gov/va/api/health/communitycareeligibility/api/CommunityCareEligibilityService.java [hunk 1]
```diff
@@ -186,7 +189,8 @@ CommunityCareEligibilityResponse search(
               description =
                   "Optional extended drive-radius to include more VA medical facilities in response"
                       + " (Does not change overall eligibility."
-                      + " Must exceed standard drive time for service-type.)")
+                      + " Must exceed standard drive time for service-type.)",
+              examples = @ExampleObject(name = "extendedDriveMin", value = "45"))
```
Comment (as-va) [inline community-care-eligibility-api/src/main/java/gov/va/api/health/communitycareeligibility/api/CommunityCareEligibilityService.java:193]: Only multiples of 10 between 10 and 90 are allowable values. This example value will not work and will break the OAS testing.

Comment (xenkidu-va) [reply community-care-eligibility-api/src/main/java/gov/va/api/health/communitycareeligibility/api/CommunityCareEligibilityService.java]: Good eye! I went ahead and updated.

**Review by taylorrassmannlibertyits — APPROVED** [review]

File: community-care-eligibility-api/src/main/java/gov/va/api/health/communitycareeligibility/api/CommunityCareEligibilityService.java (hunk 1 above)
Comment (as-va) [reply community-care-eligibility-api/src/main/java/gov/va/api/health/communitycareeligibility/api/CommunityCareEligibilityService.java]: Oops, one more note on this - the extended drive time should be greater than the default drive time. Default for primary care is 30 mins, specialty care is 60 mins. This example is using `Gastroenterology` for the service type, which is specialty care, so the extended drive time should be greater than 60.

Comment (xenkidu-va) [reply community-care-eligibility-api/src/main/java/gov/va/api/health/communitycareeligibility/api/CommunityCareEligibilityService.java]: Sounds good, updated to 70.

Comment (as-va) [reply community-care-eligibility-api/src/main/java/gov/va/api/health/communitycareeligibility/api/CommunityCareEligibilityService.java]: Thanks, I'm done griping now!

**Review by as-va — APPROVED** [review]

**Pull request merged**